frontend/src/view/UserComment/UserComment.js

```diff
@@ -0,0 +1,37 @@
+import {
+  StyledCommentContainer,
+  StyledLeftSection,
+  StyledRightSection,
+  StyledUserName,
+  StyledContent,
+  StyledDate,
+} from './UserComment.styled';
+import { useState } from 'react';
+import { UserAvatar } from '../UserAvatar/UserAvatar';
+
+export const UserComment = ({ comment }) => {
+  const [userComment, setComment] = useState(comment);
```

Comment:
Nie zostawiaj na przyszłość. Jeśli nie używamy tutaj setComment to jest to mylące. Pytanie, czy będziemy zmieniać tutaj,  czy może po zmianie będzie pobierana informacja na do listy komentarzy, która ponownie będzie renderowała komentarze. Więc zmiany na poziomie "wewnątrz komentarza" nie będzie. Nie wiem, czy zrozumiale to napisałem :)